### PR TITLE
Lavaland Eternal Part 2 the fucking bot closed my PR for being stale

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Basically just the more controversial changes of lavaland eternal.

Adds a temporary berserk martial art to replace the mayhem in a bottle. The martial art is extremely powerful in melee but effective worthless at range.

Distilled dragon's blood. Essentially just dragon's blood without RNG, you select your reward. Can only be gained through enraged drake.

Amulet of kings. Essentially just a fancy memento mori, does nothing that the memento does not. Gained from enraged drake.

Additional: Added super organs as a possible outcome of both normal dragon blood and distilled dragon blood. Essentially lungs that let you breathe anywhere and a heart that ocasionally injects epinephrine in you if you're in crit.

Replaced the spellblade with the crucible, simply put the spellblade has no balance checks at all - it delimbs without any checks for armor, on top of being ranged and autocharging and deflecting 50% of all attacks.

Candy no longer gives pure bloodcrawl as loot. You get a nerfed version instead - this nerfed version has a 15 sec cooldown per use, requires large pools of blood to be cast, and will take away 10% of your maximum blood volume whenever you enter a pool of blood.

Super shotgun hook is slightly faster at transporting you to your target and stuns them for half a second.

## Why It's Good For The Game

cool lootz

## Changelog
:cl:
add: Amulet of kings and distilled dragon blood as enraged drake rewards
balance: Replaced bottle of mayhem with berserk rune as bubblegum loot.
balance: Replaced spellblade with crucible as bubblegum loot.
balance: Candy no longer gives out pure bloodcrawl as loot - instead, you get a nerfed version.
balance: Super shotgun hook is now faster, now momentarily stuns your target and deals very minor (5 brute) damage.
/:cl:

### CODE REVIEW

**Comment by Ghommie** [comment]
What about the fature freeze? #2721

**Comment by BobJoga** [comment]
> What about the fature freeze? #2721

This was done before the feature freeze. Like the title implies it was closed for being stale.

File: modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -68,7 +68,25 @@
 	new /obj/item/clothing/suit/space/hostile_environment(src)
 	new /obj/item/clothing/head/helmet/space/hostile_environment(src)
 	new /obj/item/borg/upgrade/modkit/shotgun(src)
-	new /obj/item/gun/magic/staff/spellblade(src)
+	var/loot = rand(1,2)
+	switch(loot)
+		if(1)
+			new /obj/item/book/granter/martial/berserk(src)
+		if(2)
+			new /obj/item/gun/magic/staff/spellblade(src)
```
Comment (jjpark-kb) [inline modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm:76]: use pick() instead.

File: modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -68,7 +68,25 @@
...
+	new /obj/item/clothing/head/helmet/space/hostile_environment(src)
+	new /obj/item/crusher_trophy/demon_claws(src)
+	var/loot = rand(1,3)
+	switch(loot)
+		if(1)
+			new /obj/item/book/granter/martial/berserk(src)
+		if(2)
+			new /obj/item/blood_contract(src)
+		if(3)
+			new /obj/item/gun/magic/staff/spellblade(src)
```
Comment (jjpark-kb) [inline modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm:89]: use pick() instead.

File: modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (BobJoga) [reply modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm:76]: the original code was like this but ok

**Comment by BobJoga** [comment]
i'll turn this into something better

**Pull request closed without merging**